src/k2/dto/K23SI.h

```diff
@@ -357,5 +358,15 @@ struct K23SITxnFinalizeRequest {
 struct K23SITxnFinalizeResponse {
     K2_PAYLOAD_EMPTY;
 };
+
+struct K23SIPushSchemaRequest {
+    Schema schema;
```

Comment:
i think the way you have Schema now, we should also specify which collection this is for. In other words, the global unique key for a schema is
<collection_id>:<schema_id>:<schema_version> (with schema_id<->schema_name being 1<->1).  Else, we need the collection name in the Schema object itself, which considering how things are structured, might be a fine idea too.